### DISCUSSION TITLE
Helix on Amazon Linux 2

### DISCUSSION THREAD

**Discussion by dejlek** [helix-editor/helix#11294] [Q&A]
It is not possible to build because of: 
```
error: failed to compile `helix-term v24.7.0 (/work/dejan/git/helix/helix-term)`, intermediate artifacts can be found at `/work/dejan/git/helix/target`

Caused by:
  package `helix-parsec v24.7.0 (/work/dejan/git/helix/helix-parsec)` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.68.2
```

The pre-built binary does not work because it is linked against newer GLIBC. It would be great if the Install document had a paragraph or two about how to install Helix on Amazon Linux 2.

**Comment by pascalkuthe** [comment]
You can ise rustup to install the latest rust version.
